tests/test_odin_sequencer.py

```diff
@@ -183,6 +186,16 @@ def test_load_with_missing_directory(shared_datadir, make_seq_manager, create_pa
     ):
         make_seq_manager(directory_path)
 
+def test_explicit_module_load(make_seq_manager, create_paths):
+    """
+    Test that a module file is loaded into the manager when the load function
+    is explicitly called.
+    """
+    manager = make_seq_manager()
+    manager.load(str(create_paths('basic_sequences.py')))
```

Comment:
Moved this test function and also casted the file path to string so that line 73 of `manager.py` gets covered in the unit tests.

Reply:
👍